src/models/dao/posts.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+class PostsDAO {
+    private PDO $db;
+    public const POSTS_TABLE = 'posts';
+
+    function __construct(PDO $db) {
+        $this->db = $db;
+    }
```

Comment:
この書き方だと、UnitTestは書きにくいと思います。
```suggestion
    function __construct(public ?PDO $db = null) {
        if (is_null($db)) {
            $this->db = connectPostgreSQL();
        }
    }
```
とすると、テストが書きやすいかなと思います。